app.rb

```diff
@@ -0,0 +1,127 @@
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @students = []
+    @rentals = []
+  end
+
+  # User interaction point UI.
+  def ui
+    puts '*****Welcome to Taas school Libary!*****'
+
+    puts 'Choose an option below to proceed:'
+    puts '1 - List all books'
+    puts '2 - List people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals '
+    puts '7 - List of books rented by a student '
+    puts '8 - Exit'
+
+    options
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def options
+    option = gets.chomp
+    case option
+    when '1'
+      list_books
+    when '2'
+      list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rentals
+    when '7'
+      list_student_books
+    when '8'
+      puts 'Thank you for using this app!'
+      exit 0
+    end
+  end
+  # rubocop:enable Metrics/CyclomaticComplexity
+
+  # Create and get a book
+  def create_book
+    print 'Enter Title: '
+    title = gets.chomp
+
+    print 'Enter Author: '
+    author = gets.chomp
+    @books << Book.new(title, author)
+    puts "#{title} by #{author}  was successfully added"
+    ui
+  end
+
+  def list_books
+    @books.map { |book| puts "Book Title: #{book.title}, Author name: #{book.author}" }
+    ui
+  end
+
+  # Create and list person.
+  def create_person
+    puts 'Enter the Name'
+    name = gets.chomp
+    puts 'Enter the age'
+    age = gets.chomp
+    @students << Student.new(age, name)
+    puts "{#name}  aged #{age}  was successfully added"
+    ui
+  end
```

Comment:
- [ ] In your implementation currently, the app only allows for creating only a student and not a teacher. From the project requirements, we're to be able to create a teacher, their age, name, and specialization but your implementation didn't capture that. Kindly implement that feature 👍 
- [ ] You are supposed to give options for creating a student and a teacher, 1 for the student and 2 for the teacher. 

**See the requirement below**

![image](https://user-images.githubusercontent.com/50941074/202845801-071cf1e6-85f5-463a-b0d0-b211a40317a4.png)